src/ucp/wireup/select.c

```diff
@@ -37,6 +37,14 @@
         continue;                                                                          \
     }
 
+#define ucp_wireup_has_fast_lane(_lane_types) \
+    (_lane_types & (UCS_BIT(UCP_LANE_TYPE_AM) | UCS_BIT(UCP_LANE_TYPE_RMA) | \
+                    UCS_BIT(UCP_LANE_TYPE_AMO) | UCS_BIT(UCP_LANE_TYPE_TAG)))
+
+#define ucp_wireup_is_fast_lane(_lane_type) \
+    ucp_wireup_has_fast_lane(UCS_BIT(_lane_type))
```

Comment:
why not inline func?
all macros must be in UPPER_CASE

Reply:
yes, IMO inline function is better

Reply:
in general inline func better than macro because less potential bugs , it's type-safe, and easier to debug code with functions than with macros